src/models/locationDao.js

```diff
@@ -0,0 +1,50 @@
+const { sqlDataSource } = require("./data-source");
+
+const getListByMovieId = async (movieId) => {
+  let whereClause;
+  if (movieId) {
+    whereClause = `WHERE m.id = ${movieId}`;
+  } else {
+    whereClause = "";
+  }
```

Comment:
```suggestion
  movieId ? whereClause = `WHERE m.id = ${movieId}` : whereClause = "";
```
삼항 연산자 활용해서 줄일 수 있을 것 같네요!